### PR TITLE
Us026 add branding to the product

### PR DESCRIPTION
**Pull request by DarraghHaughey-Kainos**

### CODE REVIEW

File: public/styles.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+body {
+    font-family: Century Gothic, sans-serif !important;
```
Comment (glenn-hamilton) [inline public/styles.css]: Using `!important` doesn't make me happy

File: views/_includes/layouts/base.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello World</title>
```
Comment (glenn-hamilton) [inline views/_includes/layouts/base.html]: Can the page title be dynamically added to the template via the controller?

File: views/_includes/layouts/base.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                </nav>
+            </div>
+        </div>
+    </header>
+
+    {% block content %}
+
+    {% endblock %}
+
+    <footer>
```
Comment (glenn-hamilton) [inline views/_includes/layouts/base.html]: The text in the footer has contrast issues

File: views/_includes/layouts/base.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        <div>
+            Contact Information:
```
Comment (glenn-hamilton) [inline views/_includes/layouts/base.html:37]: The placeholder contact information can be replaced with the real information from https://www.kainos.com/contact-us/uk-and-ireland/belfast

File: views/_includes/layouts/base.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        </div>
+        <div>
+            Email: email@email.com
```
Comment (glenn-hamilton) [inline views/_includes/layouts/base.html]: To make this interactable you can use `<a href="mailto:example@email.com">example@email.com</a>`

File: views/_includes/layouts/base.html [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+        </div>
+        <div>
+            Phone: (028)123 456 789
```
Comment (glenn-hamilton) [inline views/_includes/layouts/base.html]: To make this interactable you can use `<a href="tel:12345678910">12345678910</a>`

File: views/_includes/layouts/base.html [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    <link rel="stylesheet" href="/public/styles.css">
+</head>
+
+<body>
+    <header class="navbar-light bg-light">
+        <div class="container">
+            <div class="row">
+                <nav class="navbar navbar-expand-lg">
+                    <a class="navbar-brand" href="/">
+                        <img src="/images/KainosLogo.png" width="150" height="50" alt="logo">
```
Comment (glenn-hamilton) [inline views/_includes/layouts/base.html]: The `alt` text here isn't that descriptive

File: views/_includes/layouts/base.html [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+                    </a>
+                    <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
```
Comment (glenn-hamilton) [inline views/_includes/layouts/base.html:21]: These menu items have no feedback to the user that you are hovering over them

File: public/styles.css (hunk 1 above)
Comment (DarraghHaughey-Kainos) [reply public/styles.css]: ah that was just from me testing it before we actually sorted my public folder not going to dist issue, removed it now :)

File: views/_includes/layouts/base.html (hunk 2 above)
Comment (DarraghHaughey-Kainos) [reply views/_includes/layouts/base.html]: Yes!

File: public/styles.css [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+footer {
+    padding-top: 10px;
+    padding-bottom: 10px;
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    /* background: radial-gradient(circle, rgb(40, 53, 131), rgb(0, 70, 49)); */
```
Comment (glenn-hamilton) [inline public/styles.css]: Is this needed any more?

File: views/Home.html [hunk 10]
```diff
@@ -0,0 +1,13 @@
+{% extends "_includes/layouts/base.html" %}
+{% block content %}
+<main class="container mt-4">
+    <div>
+        Home Page
+        <ul>
+            <li>
+                <a href="/hello-world">Hello World</a>
```
Comment (glenn-hamilton) [inline views/Home.html:8]: This link has contrast issues

File: views/_includes/layouts/base.html [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+            Contact Information:
+        </div>
+        <div>
+            Address: 
+            4-6 Upper Cres,
+            Belfast,
+            BT7 1NT
+        </div>
+        <div>
+            Email: <a href="mailto:email@kainos.com">email@kainos.com</a>
```
Comment (glenn-hamilton) [inline views/_includes/layouts/base.html:46]: You have an inconsistency with the hover on these links. The Email/Phone links have a blue hover, whereas the rest are remain black on hover

Comment (DarraghHaughey-Kainos) [reply views/_includes/layouts/base.html:46]: fair point

**Review by glenn-hamilton — APPROVED** [review]

**Review by steveh2707 — APPROVED** [review]
best branding ive ever seen!

**Pull request merged**